gitlint/version.py

```diff
@@ -0,0 +1 @@
+__VERSION__ = '0.0.9'
```

Comment:
Could you also add this change to the release notes in the readme file and if you want, give yourself credits for it.

Reply:
Done.